Gemfile

```diff
@@ -1,37 +1,43 @@
 source "https://rubygems.org"
 
-ruby "2.4.1"
+ruby RUBY_VERSION
```

Comment:
d'on surt aquest RUBY_VERSION?
Hauríem de forçar la mateixa que força Decidim. Per exemple en un .ruby-version

Reply:
s'està agafant del .ruby-version :smiley: 
En aquesta versió de Decidim és 2.4.1